.travis.yml

```diff
@@ -1,5 +1,13 @@
 sudo: false
 language: java
-jdk: 
+env:
+  global:
+    - AWS_ACCESS_KEY_ID=AKIAJIZZTZIXJIFQQ2WA
+    - secure: "b1v74SOUrdIn/KkPFJ+2UZ8u9qoJxahwEA7yjUWT+XIYQIipzXKUfJvuTPtmKGdhxnaqY5zI7C9rzE775pKBNG4IGMmRwRvmoxatc7UjunQ7o1jre8MnitPT3aMIZU5flXb/gJ3if5RHoVpAeDtT1FUMgt1pZtVjh0ldcrh9oRc="
```

Comment:
I'm not familiar with this. I assume it's setting `AWS_SECRET_ACCESS_KEY`. Is it safe to commit this?